src/simulation/simulation_store.jl

```diff
@@ -4,26 +4,6 @@ const STORE_CONTAINER_VARIABLES = :variables
 const STORE_CONTAINERS =
     Set((STORE_CONTAINER_DUALS, STORE_CONTAINER_PARAMETERS, STORE_CONTAINER_VARIABLES))
 
-"""
```

Comment:
I'm not sure why you moved this, but the filename now seems odd.

Reply:
I moved it to the abstract types definition file to be able to define the DecisionModel before the simulation code.